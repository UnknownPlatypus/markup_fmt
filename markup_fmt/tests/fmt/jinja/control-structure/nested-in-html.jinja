<div class="contact-info-column">
    {% if contact.email %}<a href="mailto:{{ contact.email }}">{{ contact.email }}</a>{% endif %}
    {% if contact.email %} <a href="mailto:{{ contact.email }}">{{ contact.email }}</a>{% endif %}
    {% if contact.email %} <a href="mailto:{{ contact.email }}"> {{ contact.email }}</a>{% endif %}
    {% if contact.email %}
        <a href="mailto:{{ contact.email }}">{{ contact.email }}</a>
    {% endif %}
</div>
<span class="contact-info-column">
    {% if contact.email %}<a href="mailto:{{ contact.email }}">{{ contact.email }}</a>{% endif %}
    {% if contact.email %} <a href="mailto:{{ contact.email }}">{{ contact.email }}</a>{% endif %}
    {% if contact.email %} <a href="mailto:{{ contact.email }}"> {{ contact.email }}</a>{% endif %}
    {% if contact.email %}
        <a href="mailto:{{ contact.email }}">{{ contact.email }}</a>
    {% endif %}
</span>