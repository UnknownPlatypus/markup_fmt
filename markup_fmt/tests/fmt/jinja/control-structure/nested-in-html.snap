---
source: markup_fmt/tests/fmt.rs
---
<div class="contact-info-column">
  {% if contact.email %}<a href="mailto:{{ contact.email }}">{{
      contact.email
    }}</a>{% endif %}
  {% if contact.email %}
    <a href="mailto:{{ contact.email }}">{{ contact.email }}</a>{% endif %}
  {% if contact.email %}
    <a href="mailto:{{ contact.email }}"> {{ contact.email }}</a>{% endif %}
  {% if contact.email %}
    <a href="mailto:{{ contact.email }}">{{ contact.email }}</a>
  {% endif %}
</div>
<span class="contact-info-column">
  {% if contact.email %}<a href="mailto:{{ contact.email }}">{{
      contact.email
    }}</a>{% endif %}
  {% if contact.email %}
    <a href="mailto:{{ contact.email }}">{{ contact.email }}</a>{% endif %}
  {% if contact.email %}
    <a href="mailto:{{ contact.email }}"> {{ contact.email }}</a>{% endif %}
  {% if contact.email %}
    <a href="mailto:{{ contact.email }}">{{ contact.email }}</a>
  {% endif %}
</span>
